lib/ansiblelint/generate_docs.py

```diff
@@ -45,18 +45,19 @@ def main():
         id_text = id_link.format(d['id'], d['classname'])
         grid.append([id_text, d['version_added'], d['shortdesc'], d['description']])
 
-    filename = '../../docs/docsite/rst/rules/default_rules.rst'
+    filename = Path(__file__).resolve().parents[2].joinpath(
+        "docs/docsite/rst/rules/default_rules.rst")
```

Comment:
I think the canonical way is
```suggestion
    filename = (
        Path(__file__).resolve().parents[2] / "docs" /
        "docsite" / "rst" / "rules" / "default_rules.rst"
    )
```